topi/tests/python/test_topi_tensor.py

```diff
@@ -84,18 +84,41 @@ def check_device(device):
     for device in ["llvm"]:
         check_device(device)
 
+def verify_vectorization(n, m, dtype):
+    def check_device(device):
+        if not tvm.runtime.enabled(device):
+            print("Skip because %s is not enabled" % device)
+            return
+        with tvm.target.create(device):
```

Comment:
add a check whether fp16 is supported here and also `verify_relu`
https://github.com/apache/incubator-tvm/blob/aaf62e47e64d592be770e915a7aa59d41eddb729/topi/tests/python/test_topi_transform.py#L57-L59